File: src/commands/contribute.ts
import { createCommand } from ':util/creators.ts';
import { replyToInteraction } from ':util/interactions.ts';

import { ApplicationCommandTypes } from 'discordeno';

createCommand({
    name: 'contribute',
    description: "🤝 Let's work together",
    type: ApplicationCommandTypes.ChatInput,

    run: async (_, interaction) => {
        const contributionString = [
            `🤝  Feel like helping out? Create an issue or pull request on GitHub:`,
            'https://github.com/imatpot/lunaro-manager',
        ].join('\n');

        await replyToInteraction(interaction, {
            content: contributionString,
        });
    },
});

File: src/commands/contribute.rs
use poise::{command, CreateReply};

use crate::types::{error::Error, poise::PoiseContext};

/// 🤝 Let's work together
#[command(slash_command, rename = "contribute")]
pub async fn run(context: PoiseContext<'_>) -> Result<(), Error> {
    context
        .send(
            CreateReply::default().content(
                [
                    "🤝  Feel like helping out? Create an issue or pull request on GitHub:",
                    "https://github.com/imatpot/lunaro-manager",
                ]
                .join("\n"),
            ),
        )
        .await?;

    Ok(())
}
